### ISSUE TITLE
Clarification on only BASIC passing

### ISSUE THREAD

**Issue by PeterNjeim** [5ec1cff/TrickyStore#44]
Just for clarity, as issue #41 was seemingly mistakenly locked, is only passing BASIC and not DEVICE:

1. Expected behavior
2. An issue with the user's device and not this module
3. An unsolvable issue with this module
4. A bug that isn't worth fixing in this module

I'd just like some clarity. I'm very appreciative of your work and am not demanding any new feature or anything like that, just wondering what the reason is. Thanks for your response in advance

**Comment by ElDavoo** [comment]
I have the same problem; I tried using PIF v16.9's fingerprint and it doesn't work, I tried an unrevoked fingerprint with and without fingerprint and I can't pass more than basic. (I basically tried every possible combination).  

But PIF works fine and I pass device with PIF, so clearly it's doing something trickystore is not doing.

**Comment by tryigit** [comment]
It was not closed by mistake.

[build-vars-spoofing](https://github.com/5ec1cff/TrickyStore?tab=readme-ov-file#build-vars-spoofing)

As long as you pass the basic test, it is a SKILL ISSUE.

For device integrity the props are checked first and if your goal is not strong you just need to have a non-banned fingerprint.
If you have the keybox you can pass the Strong test even using your own fingerprint, just remove the file. However, this may not be the case with custom roms and roms that have been modified with prop (I personally will never understand those who use custom roms).

Simple.

**Comment by PeterNjeim** [comment]
Thank you for your input. I indeed use a custom ROM and will look into this more deeply. I wasn't saying that it was mistakenly closed, it's fine to close it, I was saying it was locked, meaning I couldn't comment and talk to others with the same issue. I appreciate your time here

**Comment by ElDavoo** [comment]
> As long as you pass the basic test, it is a SKILL ISSUE.

I've tried other people's FPs that work on their phones and it doesn't work on mine. I've tried those same fingerprints with PIF and they pass the device test.

I also cannot pass strong with an unrevoked keybox.

We as users are supposed to just edit/replace keybox.xml and spoof files, it's a simple task and I've done it correctly, what kind of skill issue can it possibly be?

My thesis: PIF is doing something that TS is not doing. Unfortunately the logs don't give any clue.
